### PR TITLE
Fix for issue #47 - double free of COL_INFO object

### PR DESCRIPTION
**Pull request by progmachine**
* Fix for issue #47 - double free of COL_INFO object due to refcount breakage during reuse of directly found object in getColumnsInfo(...).
* One more possible memory leak of COL_INFO objects in getColumnsInfo(...).

### CODE REVIEW

File: parse.c [hunk 1]
```diff
@@ -882,6 +882,8 @@ getColumnsInfo(ConnectionClass *conn, TABLE_INFO *wti, OID greloid, StatementCla
 		if (coli_exist)
 		{
 			/* We have ready to use coli object. Cleaning it. */
+			tmp_refcnt = coli->refcnt; /* If we found coli with greloid, then some TABLE_INFO objects may have references to it -> save refcnt for them. */
+			tmp_refcnt--; /* Down the road we will increase refcnt again to account for the refernce from ConnectionClass object to coli object. */
```
Comment (davecramer) [inline parse.c:886]: spelling mistake "refernce"  in comment

**Comment by davecramer** [comment]
@progmachine Thankyou

**Comment by progmachine** [comment]
How do you think, do we need to make the 16.0.6 bugfix release?

**Comment by davecramer** [comment]
Yes, I am going to release 16.0.6 and then fast follow with 17 today

**Pull request merged**